### ISSUE TITLE
Using data_object.update() breaks unrelated queries

### ISSUE THREAD

**Issue by clharman** [weaviate/weaviate#1918]
When I use the python client's data_object.update() method to populate a previously empty property, queries including both "where" filter and "near_text" on unrelated fields stop working. Queries with either but not both of those conditions still work.

weaviate-client==3.3.0
weaviate 1.11.0

`curl -o docker-compose.yml "https://configuration.semi.technology/v2/docker-compose/docker-compose.yml?enterprise_usage_collector=false&gpu_support=false&media_type=text&ner_module=false&qna_module=false&runtime=docker-compose&spellcheck_module=false&text_module=text2vec-transformers&transformers_model=sentence-transformers-multi-qa-MiniLM-L6-cos-v1&weaviate_version=v1.11.0"`

`docker-compose up`

```
import weaviate
client = weaviate.Client("http://localhost:8080")

## Create schema
schema = {
    "classes": [
        {
            "class": "Source",
            "description": "A passage from a document",
            "properties": [
                {
                    "dataType": [
                    "text"
                    ],
                    "moduleConfig": {
                        "text2vec-transformers": {
                        "skip": False,
                        "vectorizePropertyName": False,
                        }
                    },
                    "description": "text on the page",
                    "name": "page_text"
                },
                {
                    "dataType": [
                    "string[]"
                    ],
                    "moduleConfig": {
                        "text2vec-transformers": {
                        "skip": True,
                        "vectorizePropertyName": False,
                        }
                    },
                    "description": "channels",
                    "name": "channels"
                },
                {
                    "dataType": [
                    "string"
                    ],
                    "moduleConfig": {
                        "text2vec-transformers": {
                        "skip": True,
                        "vectorizePropertyName": False,
                        }
                    },
                    "description": "screenshot path",
                    "name": "website"
                }
            ]
        }
    ]
}
client.schema.create(schema)
```

```
## Add a couple records
obj1 = {
    'page_text': 'I love a good bean burrito on a hot summer day.',
    'channels': ['Twitter'],
}
obj2 = {
    'page_text': "I don't write a lot of code, but I do like to write a lot of code.",
    'channels': ['Facebook'],
}
client.data_object.create(obj1, "Source")
client.data_object.create(obj2, "Source")
```

Similarity search on page_text with a filter on channels works as expected. Website is null as expected.
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/6020766/163451966-65b9a1e6-b5f4-43b7-8f0a-cf542d94e3b8.png">

```
## Populate the website property
query_result = client.query\
      .get("Source", ["_additional{id}", "website"])\
      .with_limit(100)\
      .do()

for source in query_result['data']['Get']['Source']:

      record_id = source['_additional']['id']

      client.data_object.update(
      data_object = {'website': 'twitter.com/jimmykimmel/beanburrito/1234'},
      class_name = 'Source',
      uuid = record_id
      )
```

Now the same query returns nothing.
<img width="803" alt="image" src="https://user-images.githubusercontent.com/6020766/163452313-6be2ab4f-fffb-442b-a505-b794d22b13c1.png">

However, removing the near_text or where clause returns records as expected.
<img width="1099" alt="image" src="https://user-images.githubusercontent.com/6020766/163452647-7994c337-8e98-4554-a210-90b7ae5d73cd.png">
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/6020766/163452715-ee290c8e-4070-4ddc-8cc0-c560b78d051a.png">

**Comment by etiennedi** [comment]
Thank you for the detailed report and the reproducing example. We'll take it from here and let you know once there are any updates.

**Comment by etiennedi** [comment]
@antas-marcin This sounds suspiciously close to the data loss bug that was fixed in `v1.14.0`. We should see if we can still reproduce this with the newest version.

**Comment by dirkkul** [comment]
this seems to be fixed, closing
